c7n/resources/elb.py

```diff
@@ -106,6 +108,34 @@ def get_resources(self, resource_ids):
                 return []
             raise
 
+    def add_tags(self, client, elbs):
```

Comment:
It's adding the tags to the result set. I see what you're saying though about how it could be confusing. I'll rename here and in RDS to clarify.